content/webapp/components/AudioPlayerNew/index.tsx

```diff
@@ -24,7 +24,7 @@ const AudioPlayerWrapper = styled(Space).attrs({
   $v: { size: 'm', properties: ['padding-top', 'padding-bottom'] },
 })<{ $isDark: boolean }>`
   background: ${props =>
-    props.$isDark ? props.theme.color('black') : props.theme.color('white')};
+    props.$isDark ? props.theme.color('black') : undefined};
```

Comment:
what's the difference between picking `props.theme.color('transparent')` (below) and `undefined` here? I _think_ I prefer the former (feels a bit more intentional?

Reply:
I thought one less line of CSS might be nice but understand what you mean so can change it back!